spec/std/tuple_spec.cr

```diff
@@ -220,6 +220,30 @@ describe "Tuple" do
     Tuple.new.last?.should be_nil
   end
 
+  it "does comparison" do 
+    tuple1 = {"a","a","c"}
+    tuple2 = {"a","b","c"}
+    (tuple1 <=> tuple2).should eq(-1)
+    (tuple2 <=> tuple1).should eq(1)
+  end 
+
+  it "does <=> for equality" do 
+    tuple1 = {0,1}
+    tuple2 = {0.0,1}
+    (tuple1 <=> tuple2).should eq(0)
+  end
+
+  it "does <=> with the same begining and different size" do
+      tuple1 = {1,2,3}
+      tuple2 = {1,2}
+      (tuple1 <=> tuple2).should eq(1)
+  end
+
+  it "does types" do
+      tuple = {1, 'a', "hello"}
+      tuple.types.to_s.should eq("{Int32, Char, String}")
+  end
```

Comment:
Couple of indentation issues here, please use `crystal tool format` to correct them.